lib/tests/streamlit/metric_test.py

```diff
@@ -187,3 +187,8 @@ def test_invalid_delta_color(self):
             "'normal', 'inverse', or 'off'",
             str(exc.exception),
         )
+
+    def test_help(self):
+        st.metric("label_test", help="   help text")
```

Comment:
Is this intentional that there are three spaces in front of `help text`?

Reply:
It's to test the dedent code too!

Reply:
ahh, ok. that makes sense.